sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -107,8 +118,8 @@ void *getOsLibraryFuncAddress(void *Library, const std::string &FunctionName);
 // environment variable.
 enum Backend { SYCL_BE_PI_OPENCL, SYCL_BE_PI_CUDA, SYCL_BE_PI_OTHER };
```

Comment:
this enum should be merged with then sycl::backend coming in https://github.com/intel/llvm/pull/1332
can be done separately